po/pt_BR.po

```diff
@@ -1048,7 +1048,7 @@ msgid ""
 "Your list of podcast subscriptions is empty. Please subscribe to some "
 "podcasts first before trying to export your subscription list."
 msgstr ""
-"Sua lista de inscrição de podcasts esta vazia, por favor se inscreva aluns "
+"Sua lista de inscrição de podcasts esta vazia, Assine por favor alguns "
```

Comment:
Is it grammatically correct to capitalize after a comma in Portuguese?  Should that be a period now instead?